modules/ROOT/pages/access-logging.adoc

```diff
@@ -117,11 +112,11 @@ If no value is found, `0` is printed instead of `-`.
 |%t
 |NCSA format of the start time of the request
 
-|%(t)W
-|The current time when the message to the access log is queued to be logged in normal NCSA format
+|%{t}W
+|NCSA format of the end time of the request
```

Comment:
for parallelism with the earlier unit clarification (line 83):

The end time of the request, in NCSA format